src/react/deckgl.js

```diff
@@ -100,12 +108,18 @@ export default class DeckGL extends React.Component {
     this._updateLayers(this.props);
 
     // Check if a mouse event has been specified and that at least one of the layers is pickable
-    const hasEvent = this.props.onLayerClick !== noop || this.props.onLayerHover !== noop;
+    const hasEvent =
+      this.props.onLayerClick !== noop ||
+      this.props.onLayerHover !== noop ||
+      this.props.onLayerDragStart !== noop ||
+      this.props.onLayerDragMove !== noop ||
+      this.props.onLayerDragEnd !== noop ||
+      this.props.onLayerDragCancel !== noop;
```

Comment:
This is an anti-React-pattern: these props are only checked once on itialization.